src/styles/blocks/product.scss

```diff
@@ -0,0 +1,51 @@
+@use '../utils/mixins' as *;
+@use '../utils/variables' as *;
+
+.product {
+  border: 1px solid $gray-border;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 408px;
+  width: 200px;
+
+  > * {
+    margin: 0 16px 16px;
+  }
+
+  &__img {
+    margin: 32px auto 40px;
```

Comment:
[Checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css): Be consistent with your margins (Add only top or bottom, but not both)